examples/test_examples/test_examples_pytest.py

```diff
@@ -0,0 +1,89 @@
+import os
+import subprocess
+import pytest
+import snapshottest
+
+
+TEST_CASES = [
```

Comment:
Maybe we can collect testcases dynamically to avoid changing this file everytime when new example is added